c7n/resources/appelb.py

```diff
@@ -485,6 +485,77 @@ def process_alb(self, client, alb):
                 alb['LoadBalancerArn'], e)
 
 
+@AppELB.action_registry.register('modify-attributes')
+class AppELBModifyAttributes(BaseAction):
+    """Modify attributes given by key/value pairs in `attributes`
+
+    Exposes the `ModifyLoadBalancerAttributes API
+    <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/elbv2.html#ElasticLoadBalancingv2.Client.modify_load_balancer_attributes>`_
+
+    :example:
+
+    .. code-block:: yaml
+
+            policies:
+              - name: turn-on-elb-deletion-protection
+                resource: app-elb
+                filters:
+                  - type: attributes
+                    key: "deletion_protection.enabled"
+                    value: false
+                actions:
+                  - type: modify-attributes
+                    attributes:
+                      "deletion_protection.enabled": "true"
+                      "idle_timeout.timeout_seconds": 120
+    """
+    schema = {
+        'type': 'object',
+        'additionalProperties': False,
+        'properties': {
+            'type': {
+                'enum': ['modify-attributes']},
+            'attributes': {
+                'type': 'object',
+                'items': {
```

Comment:
this is not valid jsonschema, please use additionalProperties=False for any object types.

Reply:
fixed, nutshell  per jsonschema spec/docs items is relevant for type: array, not for type object, properties is relevant for type:  object